config/config.docker.yml

```diff
@@ -1,169 +1,170 @@
-    # donorbox:
-    #     email: you@domain.com
-    #     key: YOUR_DONORBOX_KEY
-
-
-    # Options for interacting with crowdin.com translation projects
-    # crowdin:
-    #     projects:
-    #         web:
-    #             identifier: inaturalistweb
-    #             key: 1234
-    #         mobile:
-    #             identifier: inaturalistios
-    #             key: 5678
-    #         seek:
-    #             identifier: seek
-    #             key: 4321
-
-    # Sendgrid-specific options if using sendgrid.com to send email
+    donorbox:
+        email: <%= ENV.fetch("INAT_DONORBOX_EMAIL") { "INAT_DONORBOX_EMAIL" } %>
+        key: <%= ENV.fetch("INAT_DONORBOX_KEY") { "INAT_DONORBOX_KEY" } %>
+
+    crowdin:
+        projects:
+            web:
+                identifier: inaturalistweb
+                key: <%= ENV.fetch("INAT_CROWDIN_WEB_KEY") { "INAT_CROWDIN_WEB_KEY" } %>
+            mobile:
+                identifier: inaturalistweb
+                key: <%= ENV.fetch("INAT_CROWDIN_MOBILE_KEY") { "INAT_CROWDIN_MOBILE_KEY" } %>
+            seek:
+                identifier: inaturalistweb
+                key: <%= ENV.fetch("INAT_CROWDIN_SEEK_KEY") { "INAT_CROWDIN_SEEK_KEY" } %>
+
     sendgrid:
-        api_key: some-api-key
+        api_key: <%= ENV.fetch("INAT_SENDGRID_API_KEY") { "INAT_SENDGRID_API_KEY" } %>
+        validation_api_key: <%= ENV.fetch("INAT_SENDGRID_VALIDATION_API_KEY") { "INAT_SENDGRID_VALIDATION_API_KEY" } %>
         # Unsubscribe groups configured on Sendgrid
         asm_group_ids:
-            # The unsubscribe group used as a default for all emails sent from
-            # the web app
-            default: 1234
-
+            # Transactional emails
+            default: 25993
+            # Emails about your account
+            account: 26500
+            # Donation campaigns
+            donation: 19306
+            # News about iNat
+            news: 19705
+            # Emails to existing donors
+            donors: 28967
+            # Emails about legacy giving
+            legacy_giving: 34067
+            # Emails about accuracy experiments
+            accuracy: 35930
+        primary_ip_pool: ip_pool_1
+        alternate_ip_pool: ip_pool_2
+
+    jwt_secret: <%= ENV.fetch("INAT_JWT_SECRET") { "INAT_JWT_SECRET" } %>
+    jwt_application_secret: <%= ENV.fetch("INAT_JWT_APPLICATION_SECRET") { "INAT_JWT_APPLICATION_SECRET" } %>
+
+    banned_emails:
+        - novartismails.com
+        - ibmmails.com
+        - dupontmails.com
+        - shoprtr.com
+        - adsagency.net
+        - hubspotmails.com
+        - temasekmail.com
+        - dailysocialpro.com
+        - poczta.pl
+        - mailclubs.info
+        - osmom.justdied.com
+        - interia.pl
+        - o2.pl
+        - wordpressmails.com
+        - hotspotmails.com
+        - proconzon.com
+        - postfreeth.com
+        - waymymails.com
+        - jelly-life.com
+        - jokermails.com
+        - yourmailhost.com
```

Comment:
Should these be checked in? Currently these are semi-secret, no?

Reply:
You are probably right. I will remove this.